docs/user-guide/api-mediation/configuration-extender-passtickets.md

```diff
@@ -172,52 +178,55 @@ You configured Zowe to use PassTickets using ACF2.
 
 Before you begin this procedure, verify that the `PTKTDATA` class and ownership for the PassTicket resource (`IRRPTAUT`) have not already been defined.
 
-1.	Update the resource descriptor table (RDT) to define the `PTKTDATA` class by entering the following commands:
+1. Update the resource descriptor table (RDT) to define the `PTKTDATA` class by entering the following commands:
 
 :::note
 The PTKTDATA resource is not a predefined class.
 :::
 
-```
+```tss
 TSS ADDTO(RDT) RESCLASS(PTKTDATA) RESCODE(n) ACLST(ALL,READ,UPDATE) MAXLEN(37) 
 ```
+
 The PTKTDATA resource is added to the RDT.
 
 :::note
 Include `RESCODE(n)` in the range of 101 to 13F to make `PTKTDATA` a prefixed resource class.
 :::
 
-2.	Assign ownership for the PassTicket resource (`IRRPTAUT`). Execute the following commands: 
-```
+2. Assign ownership for the PassTicket resource (`IRRPTAUT`). Execute the following commands:
+
+```tss
 TSS ADDTO(department) PTKTDATA(IRRPTAUT) 
 ```
-3. Define PassTicket for application ID _applid_ without replay protection.
 
-```
+3. Define PassTicket for application ID _applid_
+
+```tss
 TSS ADDTO(NDT) PSTKAPPL(<applid>) SESSKEY(<key-description>) SIGNMULTI
```

Comment:
SIGNMULTI should be removed, SIGNMULTI is for 'no reply protection'

Reply:
Thanks, updated